R/fringe_io.R

```diff
@@ -51,7 +52,10 @@ fringe_write_json <- function(x, path = "", overwrite_dic = FALSE, preview_nrows
 
   y <- fringe_meta(x)
 
-  l <- list(info = y, data = d, dic = dic, preview = head(preview, preview_nrows))
+  l.meta <- list(info = y, dic = dic, preview = head(preview, preview_nrows))
+  l <- list(data = d)
```

Comment:
Include here also the dictionary. Make sure you remove the hdType column from this dictionary as it makes sense to us but not the end user

Reply:
So the dictionary should be in both `l` and `l.meta`, but in `l.meta` it shouldn't have the hdType column?

Reply:
In both yes. hdType will be on the meta,  not in the other